main/zoe/guide/contracts/atomic-swap.md

```diff
@@ -8,21 +8,18 @@ securely trade one kind of eright for another kind, leveraging Zoe for
 escrow and offer safety. At no time does any user have the ability to
 behave opportunistically.
 
-## Public Swap
+## Atomic Swap
 
-In the `publicSwap` contract, anyone who has access to the swap instance can
+In the `atomicSwap` contract, anyone who has access to the swap instance can
```

Comment:
Ah, I think I forgot to explain this part. When we changed `publicSwap` to `atomicSwap`, we made it invite only, so it's no longer open to the public. 

Let's change this to something like:
```suggestion
In the `atomicSwap` contract, anyone can securely swap with a counterparty by escrowing digital assets with Zoe and sending an invite to the swap to the counterparty.
```